.eslintrc

```diff
@@ -1,3 +1,3 @@
 {
-    "extends": "girder"
+  "extends": "girder"
```

Comment:
Why are you changing the spacing on this?  Our js files we use 4 spaces; shouldn't we do the same on json files for consistency?